web-modules/pom.xml

```diff
@@ -25,12 +25,12 @@
         <module>javax-servlets-2</module>
         <module>jee-7</module>
         <module>jooby</module>
```

Comment:
add a comment here <!-- can't upgrade to JDK 17 -->, add the same comment against restx

Reply:
done